README.md

```diff
@@ -43,11 +57,13 @@ Currently supported relations are
 
 - [Grafana](https://github.com/canonical/grafana-operator)
```

Comment:
Needs to document the relation interfaces, or provide a link to an INTEGRATION markdown file or so

Reply:
Ack. Will do. Created Jira ticket OP-232.